cdap-docs/build.sh

```diff
@@ -30,11 +30,13 @@ source _common/common-build.sh
 
 BUILD_TEMP="build-temp"
 COMMON="_common"
+COMMON_HTACCESS="$COMMON/htaccess"
 COMMON_IMAGES="$COMMON/_images"
 COMMON_SOURCE="$COMMON/_source"
 COMMON_CONF_PY="$COMMON/common_conf.py"
 COMMON_HIGHLEVEL_PY="$COMMON/highlevel_conf.py"
 COMMON_PLACEHOLDER="$COMMON/_source/placeholder_index._rst"
+HTACCESS="htaccess"
```

Comment:
A ".htaccess" is used to control the web server, and is what placed in a directory controls the behavior. Since the file would be invisible, I store it as "htacess" and then rename it when I make the zip. Google "htaccess" for all the gory details.